trp_project_scrum/security/ir_rules.xml

```diff
@@ -0,0 +1,157 @@
+        <field name="perm_read" eval="True" />
+        <field name="perm_create" eval="True" />
+    </record>
+
+    <record id="rule_project_scrum_meeting_external_scrum_users" model="ir.rule">
+        <field name="name">Show Scrum Meetings for projects they follow</field>
+        <field name="model_id" ref="project_scrum.model_project_scrum_meeting"/>
+        <field name="domain_force">[
+            ('project_id.message_partner_ids', 'in', user.partner_id.ids )]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_read" eval="True" />
+    </record>
+
+    <record id="rule_project_scrum_test_external_scrum_users_followers" model="ir.rule">
+        <field name="name">Show write and create Test Cases from projects they follow</field>
+        <field name="model_id" ref="project_scrum.model_project_scrum_test"/>
+        <field name="domain_force">[('project_id.message_partner_ids', 'in', user.partner_id.ids )]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_write" eval="True" />
+        <field name="perm_read" eval="True" />
+        <field name="perm_create" eval="True"/>
+    </record>
+
+    <record id="rule_hr_employee_external_scrum_users" model="ir.rule">
+        <field name="name">Show them their employees (if any)</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="domain_force">[('user_id', '=', user.id)]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_read" eval="True" />
+    </record>
+
+    <record id="rule_project_project_external_scrum_users" model="ir.rule">
+        <field name="name">projects readonly for external users</field>
+        <field name="model_id" ref="project.model_project_project"/>
+        <field name="domain_force">[('message_partner_ids', 'in', user.partner_id.ids)]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_read" eval="True"/>
+    </record>
+
+    <record id="rule_ir_sequence_external_scrum_users" model="ir.rule">
+        <field name="name">Customer: Read access to sequences</field>
+        <field name="model_id" ref="base.model_ir_sequence"/>
+        <field name="domain_force">[('code', 'in', ['project.task', 'user.story'])]</field>
+        <field name="groups" eval="[(4,ref('trp_project_scrum.external_scrum_users'))]"/>
+        <field name="perm_read" eval="True" />
+    </record>
+
+    <record id="rule_ir_attachments_external_scrum_users" model="ir.rule">
+        <field name="name">Read and create attachments that are on model project, task or user story</field>
+        <field name="model_id" ref="base.model_ir_attachment"/>
+        <field name="domain_force">[('res_model', 'in', ['project.project', 'user.story' , 'project.task'])]</field>
```

Comment:
this has unexpected side effects. Check out how I changed the rule locally to make this work (js+css is cached as attachment too)

Reply:
actually, reading the v10 implementation of https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_attachment.py#L356 we don't need this rule after all it seems